monai/bundle/properties.py

```diff
@@ -189,6 +185,12 @@
         BundleProperty.REQUIRED: True,
         BundlePropertyConfig.ID: "inferer",
     },
+    "dataset_data": {
+        BundleProperty.DESC: "data source for the inference / evaluation dataset.",
+        BundleProperty.REQUIRED: False,
+        BundlePropertyConfig.ID: f"dataset{ID_SEP_KEY}data",
+        BundlePropertyConfig.REF_ID: None,  # no reference to this ID
```

Comment:
could you also update the docstring here https://github.com/Project-MONAI/MONAI/blob/bca12fa2b0b1f9e45d50e6f723a5a509ee5e5a0d/monai/utils/enums.py#L669

Reply:
Sure, Thanks @wyli !